apps/chat-e2e/src/tests/chatApi/arithmeticRequest.test.ts

```diff
@@ -60,7 +60,7 @@ for (const modelToUse of modelsForArithmeticRequest) {
       const conversation = conversationData.prepareModelConversation(
         0,
         modelToUse.isSysPromptAllowed
-          ? 'Answer arithmetic question. The answer should be number, do not use natural language'
+          ? 'Compute arithmetic expression. The answer should be a number, do not use natural language'
```

Comment:
It is better to use
"Evaluate the given arithmetic expression and return only the numerical result. Do not include any explanations, units, or additional text. Provide the answer in its simplest form, using scientific notation for very large or small numbers when appropriate. Support basic arithmetic operations (+, -, *, /), exponents (^), parentheses, and common mathematical functions (sqrt, sin, cos, tan, log, ln). Round the result to 6 decimal places if necessary."
![image](https://github.com/user-attachments/assets/3d362b2d-bb70-416a-a19c-f8c1e7f14b2d)
![image](https://github.com/user-attachments/assets/58a6d37a-160c-4f44-a64b-50863cd94a52)

First screenshot is your sys.prompt
Second - the one I suggest

Reply:
done